lib/Conversion/PolyToStandard/PolyToStandard.cpp

```diff
@@ -54,8 +54,8 @@ struct ConvertFromTensor : public OpConversionPattern<FromTensorOp> {
   LogicalResult matchAndRewrite(
       FromTensorOp op, OpAdaptor adaptor,
       ConversionPatternRewriter &rewriter) const override {
-    // FIXME: If the coeffs (the underlying tensor) has a smaller dimension and
-    // we intend to embed into the ring, we must handle the types here.
+    // Note: This currently requires that the underlying tensor that holds the
+    // coeffs has the same dimension as the ring.
```

Comment:
BTW, I changed this behavior in https://github.com/google/heir/pull/152/files#diff-439000080211781bbe504b2ce568ebf0aa9b638f1ff3d82549da7347b1da6ddfR19

Now the input tensor may have smaller shape than the polynomial degree. So I think that would imply you need an extra op to zero-extend the tensor... is this doable with a `tensor.pad`? Not sure... If you agree, let's file an issue for it and keep going.

Reply:
Fixed in https://github.com/google/heir/pull/158, I'll rebase this PR once that's submitted first.